src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -29,6 +29,15 @@
           With an active mgmt-gateway service, the dashboard will continue to be served on {{currentURL}}:{{port}} and all other services will be accessible from {{currentURL}}:{{port}}/service_name
         </cd-alert-panel>
 
+        <cd-alert-panel *ngIf="serviceForm.controls.service_type.value === 'smb'"
+                        type="warning"
+                        spacingClass="mb-3"
+                        i18n>
+          SMB service management is intended for advanced users only.
+          For most scenarios, it is  recommended to use the SMB module instead.
+          To manage SMB clusters and shares, please visit the <a routerLink="/cephfs/smb">SMB page</a>.
+        </cd-alert-panel>
```

Comment:
I think @rivkap @bhkaur25 can check here and approve.
Thanks

Reply:
This text was provided by @bhkaur25